README.md

```diff
@@ -491,18 +543,32 @@ Features:
 
 * *job_fork_env*: sets the default fork environment of a job
 
+* *delete_task*: deletes a task from the workflow
+
 * *job*: creates/renames the job
 
 * *draw_job*: plot the workflow
 
 * *write_dot*: writes the workflow in .dot format
 
+* *import_dot*: imports the workflow from a .dot file
+
 * *submit_job*: submits the job to the scheduler
 
 * *get_result*: gets and prints the job results
 
+* *get_job_result*: gets and prints the job results
+
+* *get_task_result*: gets and prints the results of a given task
+
+* *print_job_output*: gets and prints the job outputs
+
+* *print_task_output*: gets and prints the outputs of a given task\n
```

Comment:
is there a reason why there is a \n only here and not in the rest of the list elements?

Reply:
Oops! I tried some stuff and I forgot to delete this one. lol